source/components/StyledArticleTagsSelector/SuggestedTags.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import noop from 'lodash/noop';
+
+import StyledTag from '../StyledTag';
+
+import TagShape from './TagShape';
+
+const Tag = styled(StyledTag)`
+    margin-right: 10px;
+    margin-top: 10px;
+`;
+
+const Wrapper = styled.div`
+    margin-top: 30px;
+    padding-top: 8px;
+`;
+
+const SuggestedLabel = styled.div`
+    color: ${({ theme }) => theme.color.fandom_dark_gray};
+    font-size: ${({ theme }) => theme.font_size.xs};
+`;
+
+function SuggestedTags({
+    className,
```

Comment:
Is this just needed so you can style it from the client ?

Reply:
Yes. And you need that if you want to use styled components via
```js
const MyComponent = styled(Component)`
  margin: 10px;
`;
```